notebooks/paper_figures/tasmax_seasonal_quantile_trend.ipynb

```diff
@@ -0,0 +1,394 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "739d29da-eb1b-4f4a-993f-692bd40df7ee",
+   "metadata": {},
+   "source": [
+    "## Notebook for tasmax's northern hemisphere summer 95th percentile maps in GDPCIR paper"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "31b7eda1-b61a-4327-9c93-6d27a8913e94",
+   "metadata": {},
+   "source": [
+    "#### last updated : 2022/11/15, by Emile Tenezakis (e.tenezakis@gmail.com). \n",
+    "#### scaling : with a remote dask cluster as a backend to the xarray datasets. With the cluster scaling parameters as is, notebook should take around 10 minutes, including normal cluster spin up time. \n",
+    "#### output : notebook saves the figure to user specific `figure_3_output_file_path` defined below.\n",
+    "#### library dependencies : This ran on rhodium's onyx environment, with the rhodium-internal `catalyst_compute_tools` and `dodola` packages pip-installed in editable mode (install `dodola` without its dependencies with the `--no-deps` flag).  \n",
```

Comment:
@emileten is there a way to produce the figures w/o the rhodium-internal packages (assuming they are not public)?

Reply:
@kemccusker actually, the only private package this relies on is catalyst_compute_tools https://gitlab.com/crs-catalyst/infrastructure/catalyst_compute_tools. I use to spin up the cluster.

Also, I'll have to change that notebook anyways given the result of our conversation yesterday. So we can pause this review until that's done and I can also switch tools and use [this](https://rhg-compute-tools.readthedocs.io/en/latest/rhg_compute_tools.html#module-rhg_compute_tools.kubernetes) instead that I think is public.